modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,94 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators validating the data
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
+being collected during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Any one of them
+is sufficient to have access to the module.
+
+ - candidate_parameter_view
+    - This permission gives the user viewing access to the module and all its tabs.
+
+ - candidate_parameter_edit
+    - This permission gives the user editing access to the module and all its tabs.
+
+## Configurations
+
+The candidate parameters module has the following configurations that affect its usage
+
+ - useProband 
+    - This setting displays/hides the Proband tab of the module.
+
+ - useFamilyID
+    - This setting displays/hides the Family tab of the module.
+
+ - useConsent 
+    - This setting displays/hides the Consent tab of the module.
+ 
+ #### Additional Parameters
+ In addition to the Settings above, it is possible to add candidate parameters to the Candidate Info page. To do so, values must be added in SQL following the example below. These queries will add a single additonal dropdown with label _"Visit plan for candidate"_, field name _"candidate_plan"_, and options _"6month"_, _"12month"_. When candidate information is entered for this field, the value is saved in the `parameter_candidate` table in SQL.
```

Comment:
"additonal" -> "additional"

The formatting of this paragraph is also screwed up and all one line, and there's a newline missing after the heading to be consistent with the other sections.

Reply:
Agreed with @driusan. A few suggestions for the formatting:
1. add a new line after the "Additional Parameters" title as suggested by @driusan 
2. move the following after the SQL example: 
These queries will add a single addit**i**onal dropdown with label "Visit plan for candidate", field name "candidate_plan", and options "6month", "12month".  Candidate information entered for this field will be saved in the `parameter_candidate` table.

Side note: I would tend to use "candidate_visit_plan" as field name instead of "candidate_plan".

Reply:
@cmadjar took your feedback into consideration and moved instructions into new file.

for the "candidate_plan" I copied it from the wiki directly and i just confirmed it works so i will not modify it in this PR